### PR TITLE
Add ArangoDB component extension fixes #1533

### LINKED ISSUE #1533: Add ArangoDB component extension

### PR DESCRIPTION
**Pull request by zbendhiba**

### CODE REVIEW

File: extensions/arangodb/runtime/pom.xml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        <version>1.1.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>camel-quarkus-arangodb</artifactId>
+    <name>Camel Quarkus :: ArangoDb :: Runtime</name>
+    <description>Perform operations on ArangoDb documents, collections and graphs.</description>
+
+    <properties>
+        <firstVersion>1.0.0-CR4</firstVersion>
```
Comment (ppalaga) [inline extensions/arangodb/runtime/pom.xml:36]:
```suggestion
        <firstVersion>1.1.0</firstVersion>
```

You probably generated the modules before 1.0.0. If you did it now, you'd get the currently correct firstVersion 1.1.0

After changing this, please rebase (the structure of the docs changed a bit) and run `mvn process-resources -Pformat`

**Review by aldettinger — APPROVED** [review]
Well done. Now, we'll now have fast startup arrangodb :)

File: extensions/arangodb/runtime/pom.xml (hunk 1 above)
Comment (zbendhiba) [reply extensions/arangodb/runtime/pom.xml]: done!

Comment (ppalaga) [reply extensions/arangodb/runtime/pom.xml]: > rebase (the structure of the docs changed a bit)

Sorry, I was wrong about this, I have not noticed that this PR was against `camel-master`. `camel-master` is yet to be reabased on top of `master` to get those changes. I'd say let's merge this and let the `camel-master` maintainer deal with the rebase conflicts.

**Review by ppalaga — APPROVED** [review]
Great work, thanks!

**Pull request merged**